osmosis-1/osmosis.zone_assets.json

```diff
@@ -2719,6 +2719,17 @@
       "path": "transfer/channel-20100/nscr",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "source",
+      "base_denom": "ibc/C97473CD237EBA2F94FDFA6ABA5EC0E22FA140655D73D2A2754F03A347BBA40B",
+      "path": "transfer/channel-8945/ibc/FC5A7360EEED0713AE3E83E9D55A69AF873056A172AC495890ACE4582FF9685A",
```

Comment:
This path is wrong, it should expand all the way to the first ibc hop's channel and base denom (never an ibc denom)

Reply:
`transfer/channel-8945/transfer/channel-1/erc20/0x091F9A57A3F58d758b6572E9d41675918EAC7F09`

Reply:
Done. Thank you.

For reference:
SRCX on Planq from BNB: erc20/0x091F9A57A3F58d758b6572E9d41675918EAC7F09
SRCX on SOURCE from Planq: ibc/FC5A7360EEED0713AE3E83E9D55A69AF873056A172AC495890ACE4582FF9685A
SRCX from SOURCE to OSMOSIS: ibc/C97473CD237EBA2F94FDFA6ABA5EC0E22FA140655D73D2A2754F03A347BBA40B